build.sbt

```diff
@@ -1,16 +1,20 @@
-val catsEffectVersion = "3.2.9"
 lazy val root = (project in file("."))
   .settings(publishOptions)
   .settings(
     name := "ciris-aws-ssm",
     organization := "com.ovoenergy",
     licenses += ("MIT", url("http://opensource.org/licenses/MIT")),
-    scalaVersion := "3.1.0",
-    crossScalaVersions := Seq(scalaVersion.value, "2.13.1", "2.12.10"),
+    scalaVersion := "3.2.0",
```

Comment:
I upgraded because the latest version of ciris demanded it (https://github.com/vlovgr/ciris/releases/tag/v2.4.0), I was getting

```
[error] class file ciris/ConfigKey.class is broken, reading aborted with class dotty.tools.tasty.UnpickleException
[error] TASTy signature has wrong version.
[error]  expected: {majorVersion: 28, minorVersion: 1}
[error]  found   : {majorVersion: 28, minorVersion: 2}
[error]
[error] This TASTy file was produced by a more recent, forwards incompatible release.
[error] To read this TASTy file, please upgrade your tooling.
[error] The TASTy file was produced by Scala 3.2.0.
```

No big deal, I've moved ciris to the previous version and put Scala back to 3.1

Reply:
I see the ciris version going back but looks like we're still on 3.2.0 in this project?